### PR TITLE
magento/devdocs#7116: Feedback on page: /guides/v2.3/install-gde/system-requirements-tech.html

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/devdocs/issues/7116

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html

## Links to Magento source code

-  ...

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ functional_areas:
   - Setup
 ---
 
+{:.bs-callout-info}
+All of the requirements described in this document are true to the latest patch release of Magento 2.3.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/system-requirements-tech.md:11]:
```suggestion
All of the requirements described here apply to the latest patch release of Magento 2.3.
```

Comment (atwixfirster) [reply src/guides/v2.3/install-gde/system-requirements-tech.md]: **done**

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**